SwrveSDK/src/google/java/com/swrve/sdk/config/SwrveConfig.java

```diff
@@ -62,4 +67,18 @@ public boolean isGAIDLoggingEnabled() {
     public void setGAIDLoggingEnabled(boolean enabled) {
         this.gAIDLoggingEnabled = enabled;
     }
+
+    /**
+     * @return if it will automatically obtain the Google Cloud Messaging Registration ID.
+     */
+    public boolean isObtainRegistrationIdEnabled() {
```

Comment:
Would a customer ever want to do both? Let us manage retrieving the id, but sometimes they want to set it?

How about calling the flag something else? isObtainRegistrationIdEnabled is a horrible api.